src/main/java/com/arcbees/website/client/application/contactform/ContactFormView.ui.xml

```diff
@@ -12,22 +12,22 @@
 
     <g:PopupPanel addStyleNames="{form.style.contactForm}" modal="true" glassEnabled="true">
         <g:HTMLPanel>
-            <form action="">
+            <form ui:field="formPanel" action="javascript:void(0)">
                 <h3>Contact us</h3>
-                <div class="">
+                <div>
                     <label>Name</label>
-                    <input type="text" name="name" autofocus="autofocus"/>
+                    <input ui:field="name" type="text" name="name" autofocus="autofocus"/>
                 </div>
-                <div class="{form.style.error}">
+                <div>
                     <label>Email</label>
-                    <input type="email" name="Email" required=""/>
+                    <input ui:field="email" name="Email"/>
```

Comment:
@meriouma `type="email"` would be required ... does it trigger browser validation?